README.md

```diff
@@ -48,6 +48,14 @@ The scripts from this repository can be used to run regression tests for public,
 
     *or*
   - Install your libraries.
+  - To test a library that is not installed in a default location like
+    `$MODELICAPATH /home/username/.openmodelica/libraries/` use
+    ```yml
+    "loadFileCommands": [
+      "loadFile(\"/path/to/package.mo\")"
```

Comment:
It can be relative, but I'm not sure to what location. I think one has to call the Python scripts from the OpenModelicaLibraryTesting directory or the scripts break anyway. Then omc is started in some sub-directory and to that location the path has to be relative.

> The way I solved this (might be useful to someone) is to keep this config file with the library in question (i.e. in its repo), and make it visible to omc by using test.py [...] --libraries ~/.openmodelica/libraries/:<path to library>

I didn't know that and it does look better. I'll add some documentation for that.

Reply:
>  one has to call the Python scripts from the OpenModelicaLibraryTesting directory or the scripts break anyway.

Yes, I had noticed that already :-P

Reply:
I'm working on come CI that uses OpenModelicaLibraryTesting and the goal is to have it running on Windows as well. When the script is working for me I'll add some CI to this repo so I don't break everything. After that one could make larger improvements.

Reply:
@AndreaBartolini, this could be interesting for you 😃

Reply:
I managed to package it up in Docker and use it in Gitea CI and locally on Windows to test a package. We can compare notes. :-)

Reply:
@bilderbuchi I published the GitHub action I was talking about at https://github.com/OpenModelica/openmodelica-library-testing-action.
Giteat is compatible with GitHub actions, so it might work for your use-case.